.github/workflows/postmerge.yml

```diff
@@ -0,0 +1,176 @@
+name: Post merge
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  THIRD_PARTY_GIT_AUTHOR_EMAIL: opensource+bot@newrelic.com
+  THIRD_PARTY_GIT_AUTHOR_NAME: nr-opensource-bot
+
+jobs:
+  generate-uuid:
+    name: Generate UUID
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+      - name: Install dependencies
+        run: cd utils && yarn install
+
+      - name: Generate UUIDs for quickstarts
+        id: generate-uuids
+        run: cd utils && yarn generate-uuids
+
+      - name: Commit changes
+        id: commit-changes
+        run: |
+          git config --local user.email "${{ env.BOT_EMAIL }}"
+          git config --local user.name "${{ env.BOT_NAME }}"
+          git add ./packs/*
+          git diff-index --quiet HEAD ./packs/* || git commit -m 'chore: generate UUID(s)'
```

Comment:
We will most likely need to add a `--skip-ci` flag to this to prevent an infinite loop of workflows